odoo/addons/test_new_api/tests/test_many2many.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from odoo.tests.common import TransactionCase
+
+
+class many2manyCase(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.ship = self.env["test_new_api.ship"].create({'name': 'Colombus'})
+        self.env["test_new_api.pirate"].create({
+            'name': 'Barbe noire',
+            'ship_ids': self.ship,
```

Comment:
This is weird and it looked like it was working as intended.
If this is forbidden, we should throw some error because I had no way to guess I was doing something wrong here.